Fix label (Address space plan)

vbusch opened the pull request:
Fixes #2778

Signed-off-by: Vanessa <vbusch@redhat.com>

k-wall commented on console/console-init/ui/src/components/MessagingInstances/MessagingInstance/Enmasse/CreateAddressSpace/Steps/Review.js:
```diff
@@ -37,7 +37,7 @@ class Review extends React.Component {
                 <FormGroup isInline label="Type" fieldId="type">
                   <Text>{newInstance.typeStandard ? 'Standard' : 'Brokered'} Address Plan</Text>
```
The word "Plan" is wrong.  It should be Space.

k-wall replied:
Update looks good, thanks @vbusch

vbusch commented:
> Your changes look good, but I notice a separate typo in the field above. It is so closely related, I feel a separate issue would be overkill.

Thank you.  Corrected it to be 'Standard address space', or 'Brokered address space'.  Also changed it to sentence-style capitalization.
